app/src/main/res/layout/activity_passport_information.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/default_margin_medium">
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/passportInfoPassportNumber"
```

Comment:
Since we're using ViewBinding, no need to add the prefix to the IDs.
```suggestion
        android:id="@+id/passportNumber"
```
Also, update the IDs below.